src/models/Forecast.js

```diff
@@ -0,0 +1,105 @@
+export default class Forecast {
+  constructor(data) {
+    this.data = data;
+  }
+
+  createForecast() {
+    const forecastSection = document.querySelector('.forecast');
```

Comment:
This way you are calling `document.querySelector` every time you create a new forecast. You have to minimize getting DOM elements

My suggestion:
```js
// index.js -> App constructor
this.forecast = new Forecast()

// then call createForecast method when you get new city
this.forecast.createForecast(data)


// Forecast.js
constructor() {
  // this way you call document.querySelector only once when creating Forecast class
  this.forecastSection = document.querySelector('.forecast')
}

createForecast(data) {
  this.forecastSection.innerHTML = '';
  ...
}
```